src/components/Form/Form.test.tsx

```diff
@@ -33,7 +33,7 @@ const SimpleForm = ({ onSubmit = () => {}, validate }: SimpleFormProps) => {
 
 interface ComplexFormValues {
   testInput: string;
-  testRadio: string;
+  testRadio: "value1" | "value2";
```

Comment:
shouldn't testSelect below also be "value1" | "value2"?

Reply:
I want to ensure both a string a union are supported which is why I am testing both for Components that take options